src/components/Layout/NavigationBar/NavigationBar.tsx

```diff
@@ -113,6 +114,13 @@ export const leftNavItems: NavigationItem[] = [
 ];
 
 export const rightNavItems: NavigationItem[] = [
+  {
+    schema: NAVIGATION_ITEM_TYPE.NavigationLabelButton,
+    id: "network",
+    label: "Select a Network",
+    path: "#",
+    customLink: <NetworkSelect />,
+  },
```

Comment:
will we implement the remove this entry after we validated a document? thereby disallowing the user to change the network once he verifies it 'screenshotting', (like what u raised the other day)

Reply:
umm... Actually I'm not sure what's the flow for this part yet. Cos apart from the dropdown, the user can also change the network from his wallet. Probably have to see what SY thinks first too.

Reply:
kk